base/actions/actions.py

```diff
@@ -709,7 +702,17 @@ async def submit(
         data.update(
             {"exposure": tracker.get_slot("exposure"), "age": tracker.get_slot("age")}
         )
+        return data
 
+    async def submit(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[Dict]:
+        """Define what the form has to do
+            after all required slots are filled"""
+        data = self.get_risk_data(tracker)
         risk = utils.get_risk_level(data)
```

Comment:
This is the main culprit - by default it looks for `symptoms_`. For the obo use case, it needs to look at `obo_symptoms_`